docs/testing/index.rst

```diff
@@ -1,14 +1,12 @@
-:orphan:
-
 .. warning::
 
     This component is an *alpha*. Interfaces may change outside of the
     normal semver policy.
```

Comment:
I want to keep a warning on each of the root pages in Unstable because there's nowhere that we explain what that word means -- it just appears as the sidebar menu heading.

I removed alpha from the title specifically because it made the menu read as
```
Unstable
 +- _testing (alpha)
 +- Experimental
```

which reads awkwardly to me.

Reply:
I'm on board to keep a warning in there, particularly for the case that someone hotlinks someone to, or is directed via a search result to, the docs.

Reply:
Works for me!